apps/mosaico/src/Mosaico.purs

```diff
@@ -174,28 +174,39 @@ articleList state setState router =
       DOM.div
         { className: "mosaico--list-article list-article-default"
         , children:
-            [ DOM.a
-                { onClick: handler_ do
-                     setState \s -> s { clickedArticle = Just a }
-                     window <- Web.window
-                     _ <- Web.scroll 0 0 window
-                     router.pushState (write {}) $ "/artikel/" <> a.uuid
+          [ DOM.a
+            { onClick: handler_ do
+                  setState \s -> s { clickedArticle = Just a }
+                  window <- Web.window
+                  _ <- Web.scroll 0 0 window
+                  router.pushState (write {}) $ "/artikel/" <> a.uuid
+            , children:
+              [ DOM.div
+                { className: "list-article-image"
+                , children:[ DOM.img { src: fromMaybe "" $ map _.url a.listImage } ]
+                }
+              , DOM.div
+                { className: "list-article-liftup"
                 , children:
-                    [ DOM.div
-                        { className: "list-article-image"
-                        , children:[ DOM.img { src: fromMaybe "" $ map _.url a.listImage } ]
-                        }
-                    , DOM.div
-                        { className: "list-article-liftup"
-                        , children:
-                            [ DOM.div
-                                { className: "mosaico--tag color-hbl"
-                                , children: [ DOM.text $ fromMaybe "" (head a.tags) ]
-                                }
-                            , DOM.h2_ [ DOM.text a.title ]
-                            ]
-                        }
-                    ]
+                  [ DOM.div
+                    { className: "mosaico--tag color-hbl"
+                    , children: [ DOM.text $ fromMaybe "" (head a.tags) ]
+                    }
+                  , DOM.h2_ [ DOM.text a.title ]
+                  , DOM.div
+                    { className: "mosaico--article--meta"
+                    , children:
+                      [ case a.premium of
```

Comment:
We could use [guard](https://pursuit.purescript.org/packages/purescript-prelude/5.0.0/docs/Data.Monoid#v:guard) here:

```purescript
guard a.premium $
  DOM.div
    { className: "mosaico--article--premium background-hbl"
    , children: [ DOM.text "premium" ]
    }
```

It will evaluate a boolean, and if it's true, the given `DOM.div` will be returned. Otherwise it will be `mempty`.

The source for `guard` is basically what you wrote here 😁
```
guard :: forall m. Monoid m => Boolean -> m -> m
guard true a = a
guard false _ = mempty
```